Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/gnucash/-/gnucash/horizontal-auto.svg)

fellen commented:
Thanks!

fellen commented:
> Thanks!

Oops, wrong PR!

@Stephan-P
```
msgfmt -c --check-accelerators="_" --statistics po/nl.po 
po/nl.po:13378: sneltoetsmarkering '_' ontbreekt in 'msgstr'
po/nl.po:31591: het aantal opmaakcodes in 'msgid' en 'msgstr' is niet gelijk
msgfmt: 2 fatale fouten gevonden
5271 vertaalde berichten, 170 vage gelijkenissen, 82 onvertaalde berichten.
```
To  understand that log, 'po/nl.po:13378' means line 13378 in nl.po—you can download that format from weblate.
That section  reads:
```
#: gnucash/gtkbuilder/dialog-account.glade:1464
msgid "Acco_unt Type"
msgstr "Rekeningsoort"
```


Can you read https://wiki.gnucash.org/wiki/Translation#Special_characters_and_other_tips and try to fix them?

Regards
Frank

Stephan-P committed:
Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
1 failing checks (0.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 97.8% (5406 of 5523 strings; 85 fuzzy)
53 failing checks (0.9%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 96.6% (5337 of 5523 strings; 121 fuzzy)
55 failing checks (0.9%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 96.5% (5333 of 5523 strings; 123 fuzzy)
55 failing checks (0.9%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/glossary/nl.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 95.9% (5299 of 5523 strings; 150 fuzzy)
56 failing checks (1.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 95.8% (5295 of 5523 strings; 154 fuzzy)
62 failing checks (1.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/glossary/nl.po: 100.0% (216 of 216 strings; 0 fuzzy)
1 failing checks (0.4%)
Translation: GnuCash/Glossary (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 95.5% (5276 of 5523 strings; 167 fuzzy)
64 failing checks (1.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/glossary/nl.po: 100.0% (216 of 216 strings; 0 fuzzy)
1 failing checks (0.4%)
Translation: GnuCash/Glossary (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 95.4% (5273 of 5523 strings; 168 fuzzy)
63 failing checks (1.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 95.4% (5271 of 5523 strings; 170 fuzzy)
68 failing checks (1.2%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Co-authored-by: Stephan Paternotte <stephan@paternottes.net>

SilentWarrior83 committed:
Translation update  by Arve Eriksson <031299870@telia.com> using Weblate

po/glossary/sv.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Swedish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/sv/

Translation update  by Arve Eriksson <031299870@telia.com> using Weblate

po/sv.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Swedish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/sv/

Translation update  by Arve Eriksson <031299870@telia.com> using Weblate

po/glossary/sv.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Swedish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/sv/

Co-authored-by: Arve Eriksson <031299870@telia.com>

brianhsu committed:
Translation update  by Brian Hsu <brianhsu.hsu@gmail.com> using Weblate

po/zh_TW.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
10 failing checks (0.1%)
Translation: GnuCash/Program (Chinese (Traditional))
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/zh_Hant/

Translation update  by Brian Hsu <brianhsu.hsu@gmail.com> using Weblate

po/zh_TW.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
10 failing checks (0.1%)
Translation: GnuCash/Program (Chinese (Traditional))
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/zh_Hant/

Co-authored-by: Brian Hsu <brianhsu.hsu@gmail.com>

MocioF committed:
Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/glossary/it.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
1 failing checks (0.1%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/glossary/it.po: 99.5% (215 of 216 strings; 0 fuzzy)
1 failing checks (0.4%)
Translation: GnuCash/Glossary (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/glossary/it.po: 99.5% (215 of 216 strings; 0 fuzzy)
1 failing checks (0.4%)
Translation: GnuCash/Glossary (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/glossary/it.po: 99.5% (215 of 216 strings; 0 fuzzy)
1 failing checks (0.4%)
Translation: GnuCash/Glossary (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 98.2% (5425 of 5523 strings; 42 fuzzy)
15 failing checks (0.2%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 97.8% (5405 of 5523 strings; 53 fuzzy)
15 failing checks (0.2%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 96.1% (5309 of 5523 strings; 134 fuzzy)
60 failing checks (1.0%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Translation update  by Giuseppe Foti <foti.giuseppe@gmail.com> using Weblate

po/it.po: 95.2% (5258 of 5523 strings; 174 fuzzy)
70 failing checks (1.2%)
Translation: GnuCash/Program (Italian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/it/

Co-authored-by: Giuseppe Foti <foti.giuseppe@gmail.com>

suciptoid committed:
Translation update  by Sucipto <sucipto@pm.me> using Weblate

po/glossary/id.po: 100.0% (216 of 216 strings; 0 fuzzy)
2 failing checks (0.9%)
Translation: GnuCash/Glossary (Indonesian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/id/

Translation update  by Sucipto <sucipto@pm.me> using Weblate

po/id.po: 88.0% (4861 of 5523 strings; 481 fuzzy)
228 failing checks (4.1%)
Translation: GnuCash/Program (Indonesian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/id/

Co-authored-by: Sucipto <sucipto@pm.me>

cyphra committed:
Translation update  by Francisco Serrador <fserrador@gmail.com> using Weblate

po/es.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
2 failing checks (0.1%)
Translation: GnuCash/Program (Spanish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/es/

Co-authored-by: Francisco Serrador <fserrador@gmail.com>

milotype committed:
Translation update  by Milo Ivir <mail@milotype.de> using Weblate

po/hr.po: 96.8% (5347 of 5523 strings; 66 fuzzy)
4 failing checks (0.1%)
Translation: GnuCash/Program (Croatian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hr/

Co-authored-by: Milo Ivir <mail@milotype.de>

Commit:
Translation update  by gallegonovato <fran-carro@hotmail.es> using Weblate

po/es.po: 100.0% (5523 of 5523 strings; 0 fuzzy)
2 failing checks (0.1%)
Translation: GnuCash/Program (Spanish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/es/

Co-authored-by: gallegonovato <fran-carro@hotmail.es>

wtuemura committed:
Translation update  by Wellington Terumi Uemura <wellingtonuemura@gmail.com> using Weblate

po/pt_BR.po: 91.6% (5062 of 5523 strings; 305 fuzzy)
131 failing checks (2.3%)
Translation: GnuCash/Program (Portuguese (Brazil))
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/pt_BR/

Translation update  by Wellington Terumi Uemura <wellingtonuemura@gmail.com> using Weblate

po/glossary/pt_BR.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Portuguese (Brazil))
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/pt_BR/

Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>

Commit:
Translation update  by Andrea Andre <andrea.tsg19@slmail.me> using Weblate

po/ro.po: 48.1% (2658 of 5523 strings; 1921 fuzzy)
740 failing checks (13.3%)
Translation: GnuCash/Program (Romanian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/ro/

Translation update  by Andrea Andre <andrea.tsg19@slmail.me> using Weblate

po/ro.po: 48.0% (2656 of 5523 strings; 1922 fuzzy)
740 failing checks (13.3%)
Translation: GnuCash/Program (Romanian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/ro/

Translation update  by Andrea Andre <andrea.tsg19@slmail.me> using Weblate

po/ro.po: 47.8% (2645 of 5523 strings; 1936 fuzzy)
742 failing checks (13.4%)
Translation: GnuCash/Program (Romanian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/ro/

Co-authored-by: Andrea Andre <andrea.tsg19@slmail.me>

fellen commented:
Thanks!
